### PR TITLE
Add mentor spotlight blog post for Agien Petra

### PR DESCRIPTION
**Pull request by TildaDares**
This PR creates a blog post page for the mentor spotlight series with Agien Petra

Screenshot 2024-01-30 at 13 55 49

Link to image: https://nextcloud.sfconservancy.org/apps/photos/api/v1/publicPreview/15296?x=2048&y=2048&token=Jx00FZsOIpRSs6pdOKEvO9MwtLFkJmDp

### CODE REVIEW

File: home/static/mentor-spotlight/agien-petra-mentor.jpg
Comment (sagesharp) [inline home/static/mentor-spotlight/agien-petra-mentor.jpg:1]: When you send a pull request with a picture, please note in the commit the original URL where the picture came from.

File: home/urls.py [hunk 1]
```diff
@@ -173,6 +173,7 @@
     re_path(r'^blog/2024-01-08/may-2024-call-for-mentoring-organizations/$', views.blog_2024_01_08_cfp_open, name='2024-01-cfp-open'),
     re_path(r'^blog/2024-01-11/outreachy-2023-in-review/$', views.blog_outreachy_2023_year_in_review, name='outreachy-2023-year-in-review'),
     re_path(r'^blog/2024-01-15/may-2024-initial-applications-open/$', views.blog_2024_01_15_initial_applications_open, name='2024-01-initial-applications-open'),
+    re_path(r'^blog/2024-03-14/mentor-spotlight-agien-petra/$', views.blog_2024_03_14_mentor_spotlight_agien_petra, name='2024-03-mentor-spotlight-agien-petra'),
```
Comment (sagesharp) [inline home/urls.py:176]: I will change the date in this path and the name field to be up-to-date when I publish this post.

File: home/views.py [hunk 2]
```diff
@@ -3029,6 +3029,9 @@ def blog_2024_01_15_initial_applications_open(request):
         'current_round': current_round,
         })
 
+def blog_2024_03_14_mentor_spotlight_agien_petra(request):
+    return render(request, 'home/blog/2024-03-14-mentor-spotlight-agien-petra.html')
```
Comment (sagesharp) [inline home/views.py:3033]: I'll have to change the blog file name and path here as well.

File: home/templates/home/blog/2024-02-04-mentor-spotlight-agien-petra.html [hunk 3]
```diff
@@ -0,0 +1,88 @@
+{% extends "base.html" %}
+{% load static %}
+{% block title %}Outreachy Mentor Spotlight: Agien Petra{% endblock %}
+{% block content %}
+  <h1>Outreachy Mentor Spotlight: Agien Petra</h1>
+  <p>
+    <i>Date: March 14, 2024</i>
```
Comment (sagesharp) [inline home/templates/home/blog/2024-02-04-mentor-spotlight-agien-petra.html:7]: I'll need to change this date when I publish the blog post.

**Review by sagesharp — COMMENTED** [review]
This looks good from an architectural perspective. You've got all the pieces for the blog post in place:
 - blog HTML template
 - URL path
 - View directing the URL to render the template
 - PseudoPage to make the blog post appear on the blog post list page

I'll pull these changes in, and test it locally. I may have some comments on the blog content later.

File: home/static/mentor-spotlight/agien-petra-mentor.jpg
Comment (TildaDares) [reply home/static/mentor-spotlight/agien-petra-mentor.jpg:1]: Noted! I added the image URL to the description for now.

**Comment by sagesharp** [comment]
This code was merged -- GitHub just doesn't recognize it as merged. Closing this PR -- good job on your first blog post, @TildaDares ! :tada:

**Pull request closed without merging**